test/evergreen.yml

```diff
@@ -952,6 +955,23 @@ tasks:
             set -o verbose
             UBSAN_OPTIONS=print_stacktrace=1:halt_on_error=1:abort_on_error=1:disable_coredump=0 ./ex_access
 
+  # Start of cppsuite test tasks.
+
+  - name: poc-test
+    tags: ["pull_request"]
+    depends_on:
+      - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/build_posix/"
+          script: |
+            set -o errexit
+            set -o verbose
+
+            ${test_env_vars|} $(pwd)/test/cppsuite/run -t poc_test -f test/cppsuite/configs/config_poc_test_default.txt -l 1
+
```

Comment:
There's a Python script `test/evergreen/evg_cfg.py` that is invoked by `dist/s_evergreen` to verify missing test for "make check" and "csuite" test types, as when a new test is added into these 2 test types, people may forget to add the test as a separate pull request task. Not sure if you'd like to treat cppsuite as a similar 3rd test type. I feel maybe not as you don't actually want every cpp test to be part of pull request testing. 

Do you mind adding a comment at the end of this code block `# End of cppsuite test tasks.` as a closing clause?

Reply:
We do not want every cppsuite test to be part of PR testing. I'll add the comment for the closing clause.